### ISSUE TITLE
(REQ) Replay with clicks

### ISSUE THREAD

**Issue by AmogusNes** [matcool/TrashBot#38]
I think replay bot is the best free bot for geometry dash. I really want to add the ability to make replays with clicks

**Comment by TechStudent10** [comment]
I totally agree with this. There is already a pull in #3 to add it, but mat for some reason doesn't want it.

**Comment by x203f** [comment]
U can, use ClickBot. Its a free bot clicker. You can play the replay with that clickbot and you will hear clicks, holds and short clicks.
